components/RoomList/index.js

```diff
@@ -15,12 +15,16 @@ export default function RoomList() {
   `;
 
   const RoomListTitle = styled.h3`
-    color: pink;
+    color: #ff0054;
   `;
   const RoomListItem = styled.li`
     list-style: none;
     font-size: 20px;
     margin-bottom: 40px;
+    background-color: #ced4da;
+    border-radius: 4px;
+    padding: 10px;
+    margin-right: 30px;
```

Comment:
Hey, styles like this you could outsource in a seperate styles.js.

This is the only thing i would change, looks good!